keepassxc-browser/background/page.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
 const defaultSettings = {
+    afterFillSorting: SORT_BY_DEFAULT,
```

Comment:
What does "Sort By Default" mean? Need to make this explicit on what we are sorting by.

Reply:
The help text under the option says: "When Relevant credential first option is selected, the credential for previously filled login is set as first in Autocomplete Menu's listing." Should we add some clarification what the default is, if that is not obvious?

Reply:
No I meant make it explicit in code. SORT_DEFAULT  is meaningless. Needs to have a better name.

Reply:
Got it.

Reply:
Not sure about the variable name. When default, everything is sorted by the "Sort matching credentials by" setting, not alphabetical or natural order. Maybe `SORT_BY_MATCHING_CREDENTIALS` could describe it better.